expo/expo-cli
# created ios command

# Why

- Created internal, undocumented `run:ios` command for dogfooding dev client.

# Test Plan

- Command has two different workflows:
  - `expo run:ios` -- build ios app onto device, this requires `expo start --dev-client` to also be run in order to start up the bundler.
  - `expo run:ios --bundler` -- build and setup bundler for dev client apps. 
- `expo run:ios -d` select connected device, should fail unless the device has been provisioned with xcode ahead of time.
- Add a package `yarn add expo-camera` then rerun `expo run:ios`, pods should automatically be installed.
- Build logs should always be written to `.expo/xcodebuild.md`
